pkgs/by-name/cq/cq/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, fetchurl
+, buildGraalvmNativeImage
+, graalvmCEPackages
+}:
+
+buildGraalvmNativeImage rec {
+  pname = "cq";
+  version = "2024.02.13-23.14";
+
+  # we need both src (the prebuild jar)
+  src = fetchurl {
+    url = "https://github.com/markus-wa/cq/releases/download/${version}/cq.jar";
+    sha256 = "sha256-JQp9fBR7jhenXcXSVMFsWosKUsjNxWVU+QnTz0X+dwk=";
```

Comment:
The convention these days is to use `hash =` instead of `sha256 =` (like you did for build-src below)

Reply:
Done